jena/src/main/resources/META-INF/resources/index.xed

```diff
@@ -94,29 +94,23 @@
               </xed:choose>
 
             </xed:bind>
-
             <xed:bind xpath="sortBy[field[@name='year'][@order='descending']]"/>
-
           </xed:bind>
 
           <button class="btn btn-sm btn-primary" type="submit" xed:target="servlet" xed:href="MCRQLSearchServlet">
             <xed:output i18n="button.search"/>
           </button>
-
         </div>
       </xed:form>
     </div>
   </article>
 
-  <xinclude
-      uri="xslStyle:response-newest:solr:q=status:confirmed+AND+year:NOW&amp;rows=4&amp;sort=year+desc,created+desc&amp;facet=false&amp;fl=id,year"/>
+  <xinclude uri="xslStyle:response-newest:solr:q=%2BpartOf:true+status:confirmed+AND+year:NOW&amp;rows=4&amp;sort=year+desc,created+desc&amp;facet=false&amp;fl=id,year"/>
```

Comment:
should be:
xslStyle:response-newest:solr:q=partOf:true+AND+status:confirmed+AND+year:NOW&amp;rows=4&amp;sort=year+desc,created+desc&amp;facet=false&amp;fl=id,year